libr/core/cmd.c

```diff
@@ -4998,6 +5000,13 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(arged_command) {
 
 	pr_args->has_space_after_cmd = !ts_node_is_null (args) && ts_node_end_byte (command) < ts_node_start_byte (args);
 	res = r_cmd_call_parsed_args (state->core->rcmd, pr_args);
+	if (res == R_CMD_STATUS_WRONG_ARGS) {
+		const char *cmdname = r_cmd_parsed_args_cmd (pr_args);
+		eprintf ("Wrong number of arguments passed to `%s`, see its help with `%s?`\n", cmdname, cmdname);
+	} else if (res == R_CMD_STATUS_ERROR) {
+		const char *cmdname = r_cmd_parsed_args_cmd (pr_args);
+		eprintf ("Something wrong during the execution of `%s` command.\n", cmdname);
```

Comment:
Aren't many commands printing their own error messages making "Something wrong during the execution of " redundant and very verbose.

Reply:
Also overlaps with https://github.com/radareorg/radare2/issues/16147

Reply:
I have used a R_LOG_DEBUG, so when you set `log.level=1` you can still see this message. I think it may be useful for debugging in some cases, but it won't show up during a regular session.